src/main/java/frc/robot/subsystems/color_wheel/commands/FindColor.java

```diff
@@ -0,0 +1,74 @@
+package frc.robot.subsystems.color_wheel.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.color_wheel.ColorWheel;
+
+/**
+ * Spin the wheel until you reach the desired color.
+ */
+
+public class FindColor extends CommandBase {
+    private final ColorWheel colorWheel;
+    private final String targetColor;
+    private final double power;
+    private String prevColor;
+    private int cutoff = 10;
+
+    public FindColor(ColorWheel colorWheel, String targetColor, double power) {
+        this.colorWheel = colorWheel;
+        this.targetColor = targetColor;
+        this.power = power;
+    }
+
+    @Override
+    public void initialize() {
+        colorWheel.updateSensor();
+        String initColor = colorWheel.getColorString();
+        switch (initColor) {
+            case "RED":
+                switch (targetColor) {
+                    case "GREEN", "BLUE" -> colorWheel.setPower(power);
+                    case "YELLOW" -> colorWheel.setPower(-power);
+                }
+                break;
+            case "GREEN":
+                switch (targetColor) {
+                    case "RED" -> colorWheel.setPower(-power);
+                    case "BLUE", "YELLOW" -> colorWheel.setPower(power);
+                }
+                break;
+            case "BLUE":
+                switch (targetColor) {
+                    case "RED", "YELLOW" -> colorWheel.setPower(power);
+                    case "GREEN" -> colorWheel.setPower(-power);
+                }
+                break;
+            case "YELLOW":
+                switch (targetColor) {
+                    case "RED", "GREEN" -> colorWheel.setPower(power);
+                    case "BLUE" -> colorWheel.setPower(-power);
+                }
+                break;
+        }
+        prevColor = initColor;
+    }
+
+    @Override
+    public void execute() {
+        colorWheel.updateSensor();
+        if (!colorWheel.getColorString().equals(prevColor)) {

```

Comment:
you don't need this if statement, `isFinished()` takes care about it

Reply:
The execute method takes care of the power cutoff. The if statement in execute checks whether the wheel moved to a different color. The if statement in isFinished checks whether the color had been found.